internal/servers/worker/proxy/options.go

```diff
@@ -0,0 +1,11 @@
+package proxy
```

Comment:
Even though there are no packages now I wanted to define the signature as these will be used shortly